include/control_toolbox/pid.hpp

```diff
@@ -113,18 +113,27 @@ class CONTROL_TOOLBOX_PUBLIC Pid
    */
   struct Gains
   {
-    // Optional constructor for passing in values without antiwindup
+    // Optional constructor for passing in values without antiwindup/save i-term
```

Comment:
```suggestion
    // Optional constructor for passing in values without antiwindup and save i-term
```

Reply:
What do you think that we change the interface to only one constructor with default values for `antiwindup` and `save_iterm`?